nixos/modules/profiles/hardened.nix

```diff
@@ -115,4 +114,8 @@ with lib;
   # Ignore outgoing ICMP redirects (this is ipv4 only)
   boot.kernel.sysctl."net.ipv4.conf.all.send_redirects" = mkDefault false;
   boot.kernel.sysctl."net.ipv4.conf.default.send_redirects" = mkDefault false;
-}
+  # Disabling this so  an attacker cannot gain root access by passing init=/bin/sh as a kernel parameter.
+  boot.loader.systemd-boot.editor = false;
+  # Coredump gives infomation (sometimes sensitive) during crash, it also slows down the system.
+  systemd.coredump.enable = false; 
+}  
```

Comment:
trailing spaces after the closing bracket

Reply:
This is my first contribution to nixpkgs and I am confused on what is wrong, could someone help me?

Reply:
Hi, sure thing.

When editing the file some spaces (' ') were left after the '}' but before the end of the line.

Those are trailing spaces.

The CI checks failed due to those here: https://github.com/NixOS/nixpkgs/actions/runs/5884407981/job/15959064340?pr=248060

![image](https://github.com/NixOS/nixpkgs/assets/303897/03986923-c91a-4f1e-8c08-e01c5a6d658a)